SysTrayIcon/explainer.md

```diff
@@ -0,0 +1,129 @@
+    * This API is to allow an installed web app to have an icon on the system tray area when the app is open/running.
+
+## Use Cases
+### Instant Messaging apps
+
+![The Teams app on Windows system tray icon menu](teamsMenu.png)
+The screenshot shows how in the menu the user can open, quit, or change the user's "status" in a messaging app. (The menu shown has several levels, which is not in scope of the API, but does show the usefulness of providing different options to the end user).
+
+![Telegram app on Windows showing the number of unread messages and a menu](telegramtray.png)
+The previous screenshot shows a different instant messaging app that displays a badge with the number of unread messages and a menu that allows you to disable notifications for itself.  
+
+### Collaboration Suites
+A user can directly access their calendar or schedule a meeting right from the app's system tray icon. Notice as well in this case the actions offered in the menu are different from the ones specified in the icon in the dock. 
+
+![The Teams app on macOS providing a quick way to schedule a meeting](teamsMacSysTray.png)
+
+## Proposed Solution
+
+The scope of this feature is to allow an installed Web app to have an icon on the system's tray area if the application is open and running.
+
+There feature consists of several parts, namely, a way for the app to indicate that it wants to have a icon on the tray area and a way to specify menu options for when the user right-clicks on the icon. 
+
+### Opting into the System Tray icon
+
+For an app to opt into having an icon on the platform's system tray area, it can use the `tray_icon` entry in the manifest field.
+
+```json
+{
+  "short_name": "webapp",
+  "name": "Awesome Web App",
+  ...,
+  "tray_icon": [{
+      "src": "/trayicon.png",
+      "sizes": "256x256",
+      "type": "image/png"
+    }]
+}
+
+```
+
+This icon will be present ONLY if the application is open and running. This icon can also be different from the one used in the platforms app launcher. Clicking on the icon will open/focus the app's window. Right-clicking on the icon will bring up the app's system tray menu.
+
+
+### System Tray Menu
+As stated before, the icon on the system tray for an installed web app will open/focus the app's window when clicked and will have a menu when the user double clicks. This menu can be specified in the manifest file, and uses the same format as the one for shortcuts.
+
+```json
+"tray_menu": [
+  {
+    "name": "Today's agenda",
+    "short_name": "Agenda",
+    "description": "View your agenda for today",
+    "url": "/today",
+    "icons": [
+      {
+        "src": "today.png",
+        "sizes": "192x192"
+        }
+    ]
+  }
+]
+```
+
+Giving the developer the option to have different menu options in the shortcuts and system tray menu allows them to provide a better UX for users that doesn't repeat unnecessary options. By default, the system tray menu should have options to open and to quit the app.
```

Comment:
NIT: The 'default' options seems like an implementation detail to be left up to the browser, rather than a part that should be included in the standard.